tests/integration/spec.bats

```diff
@@ -66,12 +66,14 @@ function teardown() {
 }
 
 @test "spec validator" {
+  TESTDIR=$(pwd)
   cd "$HELLO_BUNDLE"
-  # note this test runs from the temporary bundle directory not the integration root
-  # note this test is brittle against specs changes that lead runc's spec command
-  # todo get the validate program, gojsonschema, and schema/*s.json from godeps?
 
   run git clone https://github.com/opencontainers/runtime-spec.git src/runtime-spec
+  [ "$status" -eq 0 ]
+
+  SPEC_COMMIT=$(grep runtime-spec ${TESTDIR}/../../Godeps/Godeps.json -A 4 | grep Rev | cut -d":" -f 2 | tr -d ' "')
+  run git -C src/runtime-spec reset --hard "${SPEC_COMMIT}"
```

Comment:
I'm not familiar with bats, but I think you want to check the exit status of the Git command with `[ "$status -eq 0]`.  The existence of `schema.json` isn't sufficient to tell if the `reset` succeeded.